### PR TITLE
rackspace: enable env var for flavor id

### PR DESCRIPTION
**Pull request by ehazlett**
Enable setting the Rackspace flavor id via environment variable.

### CODE REVIEW

**Comment by ehazlett** [comment]
@ggiamarchi would you mind reviewing this?  thanks!

File: drivers/rackspace/rackspace.go [hunk 1]
```diff
@@ -76,12 +76,12 @@ func GetCreateFlags() []cli.Flag {
...
-			Value: "",
 		},
 		cli.StringFlag{
-			Name:  "rackspace-flavor-id",
-			Usage: "Rackspace flavor ID. Default: General Purpose 1GB",
-			Value: "general1-1",
+			Name:   "rackspace-flavor-id",
+			Usage:  "Rackspace flavor ID. Default: General Purpose 1GB",
+			Value:  "general1-1",
+			EnvVar: "RACKSPACE_FLAVOR_ID",
```
Comment (ggiamarchi) [inline drivers/rackspace/rackspace.go]: The others env var names have the OpenStack prefix `OS_`. I think we should use the same rule here (`OS_FLAVOR_ID`) for naming consistency.

Comment (ggiamarchi) [inline drivers/rackspace/rackspace.go]: This env var should be added in the documentation

**Comment by ehazlett** [comment]
@ggiamarchi thanks!  updated!

**Comment by ggiamarchi** [comment]
LGTM

**Pull request merged**